playground/protocol-kit/deploy-safe.ts

```diff
@@ -15,7 +15,7 @@ interface Config {
 }
 
 const config: Config = {
-  RPC_URL: 'https://sepolia.gateway.tenderly.co',
+  RPC_URL: 'https://rpc.sepolia.org',
```

Comment:
Why changed the RPC? If having issues with the tenderly one we should change it across the repo